index.bs

```diff
@@ -5219,6 +5226,17 @@ type.
         "Promise" "&lt;" ReturnType "&gt;"
 </pre>
 
+<pre class="grammar" id="prod-OpenDictionaryType">
+    OpenDictionaryType :
+        "OpenDictionary" "&lt;" Type OpenDictionaryTypeMaybeValue
```

Comment:
It seems fine to me if you can also typedef the thing. We don't have lots of these and first argument being optional is weird.